### PR TITLE
Describe how to filter on nested properties with GraphQL

### PR DESCRIPTION
**Pull request by antograssiot**
Follows https://github.com/api-platform/core/pull/1868

### CODE REVIEW

File: core/filters.md [hunk 1]
```diff
@@ -508,7 +508,7 @@ use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\SearchFilter;
 /**
  * @ApiResource
  * @ApiFilter(OrderFilter::class, properties={"product.releaseDate"})
- * @ApiFilter(SearchFilter::class, properties={"product.name": "exact"})
+ * @ApiFilter(SearchFilter::class, properties={"product.color": "exact"})
```
Comment (dunglas) [inline core/filters.md]: Why this change?

File: core/graphql.md [hunk 2]
```diff
@@ -68,3 +68,67 @@ class Offer
...
+{
+    // ...
+}
+```
+
+
+The above allows you to find offers by their respective product's color like for the REST Api.
+You can then filter using the folowing syntax:
+
+```
```
Comment (dunglas) [inline core/graphql.md]: you can use ```graphql do enable syntax highlighting

File: core/graphql.md [hunk 3]
```diff
@@ -68,3 +68,67 @@ class Offer
...
+        }
+      }
+    }
+  }
+}
+```
+
+Or order your results like:
+
+```
```
Comment (dunglas) [inline core/graphql.md]: sale here

**Comment by antograssiot** [comment]
all set as requested @dunglas

**Comment by dunglas** [comment]
Thanks @antograssiot
